subprojects/dependency-management/src/integTest/groovy/org/gradle/integtests/resolve/transform/ArtifactTransformCachingIntegrationTest.groovy

```diff
@@ -205,10 +205,10 @@ class ArtifactTransformCachingIntegrationTest extends AbstractHttpDependencyReso
         then:
         output.count("> Transform lib1.jar (project :lib) with MakeBlueToGreenThings") == 1
         output.count("> Transform lib2.jar (project :lib) with MakeBlueToGreenThings") == 1
-        output.count("> Transform lib1.jar.green with MakeGreenToYellowThings") == 1
-        output.count("> Transform lib2.jar.green with MakeGreenToYellowThings") == 1
-        output.count("> Transform lib1.jar.green with MakeGreenToRedThings") == 1
-        output.count("> Transform lib2.jar.green with MakeGreenToRedThings") == 1
+        output.count("> Transform lib1.jar (project :lib) with MakeGreenToYellowThings") == 1
+        output.count("> Transform lib2.jar (project :lib) with MakeGreenToYellowThings") == 1
+        output.count("> Transform lib1.jar (project :lib) with MakeGreenToRedThings") == 1
+        output.count("> Transform lib2.jar (project :lib) with MakeGreenToRedThings") == 1
```

Comment:
This doesn't look right... We should at least display the file name that we are transforming (`lib1.jar.green`) instead of the original name (`lib1.jar`).

Reply:
We can't, since we don't know the file(s) from the previous step, yet. We could join them together, though. The other option would be to show the chain of transforms at the end/beginning.